### PR TITLE
ENH: Increase `itk::DiscreteGaussianCurvatureQuadEdgeMeshFilter` coverage

### PR DESCRIPTION
**Pull request by jhlegarreta**
- STYLE: Use medial capitals to name variables in test
- ENH: Increase itk::DiscreteGaussianCurvatureQuadEdgeMeshFilter coverage

## PR Checklist
- [X] No [API changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#breaking-changes) were made (or the changes have been approved)
- [X] No [major design changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#design-changes) were made (or the changes have been approved)

### CODE REVIEW

File: Modules/Filtering/QuadEdgeMeshFiltering/test/itkDiscreteGaussianCurvatureQuadEdgeMeshFilterTest.cxx [hunk 1]
```diff
@@ -58,19 +58,20 @@ itkDiscreteGaussianCurvatureQuadEdgeMeshFilterTest(int argc, char * argv[])
...
   using WriterType = itk::MeshFileWriter<MeshType>;
   auto writer = WriterType::New();
   writer->SetInput(output);
   writer->SetFileName("gaussian_curvature.vtk");
   writer->Update();
 
-  // ** PRINT **
-  std::cout << gaussian_curvature;
+
+  std::cout << gaussianCurvatureFilter;
```
Comment (dzenanz) [inline Modules/Filtering/QuadEdgeMeshFiltering/test/itkDiscreteGaussianCurvatureQuadEdgeMeshFilterTest.cxx]: Should there be "Test finished" message here?

**Review by dzenanz — APPROVED** [review]

File: Modules/Filtering/QuadEdgeMeshFiltering/test/itkDiscreteGaussianCurvatureQuadEdgeMeshFilterTest.cxx (hunk 1 above)
Comment (jhlegarreta) [reply Modules/Filtering/QuadEdgeMeshFiltering/test/itkDiscreteGaussianCurvatureQuadEdgeMeshFilterTest.cxx]: Done here: https://github.com/InsightSoftwareConsortium/ITK/pull/4286/files#diff-bf9342a65397b9400ab9a8de8e41bd5f0b03dd695630d86cb7e15dc5d53905edR76

**Review by thewtex — APPROVED** [review]
:100:

**Comment by dzenanz** [comment]
Please rebase.

**Pull request merged**